app/assets/javascripts/test/puppeteer/dataset_rendering.screenshot.js

```diff
@@ -69,19 +69,30 @@ const datasetNames = [
   "confocal-multi_knossos",
   "fluro-rgb_knossos",
   "dsA_2",
+  "2017-05-31_mSEM_scMS109_bk_100um_v01-aniso",
+  "ROI2017_wkw_fallback",
 ];
 
 const viewOverrides: { [key: string]: string } = {
   e2006_knossos: "4736,4992,2176,0,1.0",
+  "2017-05-31_mSEM_scMS109_bk_100um_v01-aniso": "4608,4543,386,0,4.00",
+  ROI2017_wkw_fallback: "#535,536,600,0,1.18",
```

Comment:
Not sure whether that is important, but in the other viewOverrides the `#` was left out.